pkg/universe.dagger.io/alpha/kubernetes/helm/upgrade.cue

```diff
@@ -79,12 +85,12 @@ import (
 	// for anything that is not covered by the struct fields
 	flags: [...string]
 
-	// used to avoid name clashes
-	let releaseName = name
-
 	run: docker.#Run & {
-		input: image.output
-		entrypoint: ["helm"]
+		// should probably run always since oftentimes
+		// there a re no changes visible to dagger,
+		// but a new release should be rolled out
+		always: true
```

Comment:
I have set this to always since it has side effects and dagger cannot really know if it should run or not based on the cache.